### ISSUE TITLE
Display connection status in VS

### ISSUE THREAD

**Issue by Adam--** [OYIon/XamarinFormsLive#59]
As a user who has an intermittent connection to the device deployed to, I'd like to know if LiveXAML is connected at a glance in Visual Studio so I don't have to search through the debug output to determine if it is connected.

What would you think of an indicator in the status bar?

**Comment by Adam--** [comment]
See also #41 Show error if xlserver.exe is not found

**Comment by ionoy** [comment]
Yep, that's a good idea. The same indicator could show both if the server is running and if the connection is active.
